shell.nix

```diff
@@ -6,5 +6,6 @@ in release.pkgs.stdenv.lib.overrideDerivation release.clay.env (oldAttrs: rec {
   nativeBuildInputs = (oldAttrs.nativeBuildInputs or []) ++ [
     release.cabal
     release.pkgs.haskellPackages.cabal2nix
+    release.pkgs.haskellPackages.ghcid
```

Comment:
Why is this needed?

Reply:
Oops, it is left from my toolchain. It is allows to watch after changes in ghci and automatically reload changed modules.

Reply:
No worries :) it should be removed from the PR though.